app/components/blacklight/response/facet_group_component.html.erb

```diff
@@ -15,7 +15,6 @@
       aria: {
         controls: @panel_id,
         expanded: 'false',
-        label: t('blacklight.search.facets.group.toggle'),
```

Comment:
Should this key be removed from our i18n files?

Reply:
It is the only place this key is used but I intentionally left it in case downstream applications utilize it in an override. Would you typically remove it here? Happy to.

Reply:
I think we need to remove it or we'll just accumulate junk forever.  At the very least ticket it for removal at some other time so it isn't forgotten.

Reply:
I have removed it. I accumulate enough junk in other areas of my life!